doc/classes/@GlobalScope.xml

```diff
@@ -1379,6 +1379,7 @@
 				    "b": 2
 				}
 				[/codeblock]
+				[b]Note:[/b] This will return [code]Signal()[/code] and [code]Callable()[/code] for those types, regardless of the value, not storing any details about the actual value.
```

Comment:
"for those types" is kind of unclear on what it refers to. Maybe something like
```suggestion
				[b]Note:[/b] Converting [Signal] or [Callable] is not supported and will result in empty value for these types, regardless of their data.
```
would be better.